tools/msp_split/splitMSP.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import os
+from typing import List
+
+from matchms.exporting import save_as_msp
+from matchms.importing import load_from_msp
+from matchms import Spectrum
+
+
+def read_spectra(filename: str) -> List[Spectrum]:
+    """Read spectra from file
+
+    Args:
+        filename (str): Path to .msp file from which to load the spectra
+
+    Returns:
+        List[Spectrum]: Spectra contained in the file
+    """
+    return list(load_from_msp(filename, False))
+
+
+def get_spectra_names(spectra):
+    return [x.get("compound_name") for x in spectra]
```

Comment:
Please also add code documentation same as in the example above for `read_spectra`

Reply:
Documentation has been added (https://github.com/RECETOX/galaxytools/pull/234/commits/86e20e2c7512b69aa59bde47775b990904a60e52)